atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1094,6 +1094,16 @@ void WebContents::StopFindInPage(content::StopFindAction action) {
   web_contents()->StopFinding(action);
 }
 
+void WebContents::ShowDefinitionForSelection() {
+#if defined(OS_MACOSX)
+  const auto view = web_contents()->GetRenderWidgetHostView();
+  if (view)
+    view->ShowDefinitionForSelection();
+#else
+  NOTIMPLEMENTED();
```

Comment:
`NOTIMPLEMENTED()` means this code path should never be executed, it is a runtime check for programming errors, when linked with Debug build of libchromiumcontent it will throw assert error.

Leaving the implementation empty would be fine here.